docs/docs/integration/supreme.md

```diff
@@ -69,253 +87,233 @@ As shown in Figure&nbsp;1, the back-end needs to be configured before being able
+<div id="config-options-example"></div>
 
-Naturally, clients and back-end need to agree on these parameters. Hence, it makes sense to set them inside
-a common module that is shared by back-end and clients. This leads to the following shared constants:
+??? example "Comprehensive example of Makoto config options"
+    The below config illustrates configuring two different Android apps: a regular one for the masses and a second one
+    with much tighter security constraints. This makes no sense when Warden Supreme is integrated into a back-end.
+    If, however, a dedicated attestation service is deployed that is then used to issue certificates for apps used by different services, this can be legitimate. 
+    A single iOS-app is configured for test purposes only. In this example, the iOS app has not yet launched and is purely simulated.
+    To still be able to test the attestation code path for iOS, custom trusted roots are set and all iOS attestation proofs
+    sent to the back-end are created in software, purely for evaluation purposes.
+    
+    **Be sure to check the annotations!**
+    
+    ```kotlin
+    --8<-- "Readme-Config.kt:15"
+    ```
+    
+    1. The basic application for the masses
+    2. A second, experimental high-security app
+    3. Different package name from the first app
+    4. Enforce minimum version, Android 16, an up-do-date security patches
+    5. Allow for more leeway
+    6. Only remote key provisioning is considered trustworthy for this app
+    7. Only the RKP trust anchor is considered trustworthy
+    8. We want our app to have a dedicated HSM
+    9. By default, Android 13 with a somewhat will be required without enforcing the most recent security patches or StrongBox to reach a wider audience.
+    This concerns the first app, since the second one overrides these values.
+    10. This is hardly used in practice and shows the default
+    11. This is rather optimistic, but the majority of devices running Android 13 should not screw this up.
+    12. Usully, you will always want hardware attestation, so you'd need to explicityl disable it
+    13. This is for devices launched with Android 7.0 who have not received an upgraded KeyMaster. Such devices are onyl capable of key attestation.  
+    Their app attestation claims are only software-atested!
+    14. Warden Supreme does not need to enforce this, because cryptographic nonces are used to ensure freshness.  
+    It is not recommended to set this value, because many OEMs mess this up!
+    15. Required if you run Warden behind a proxy to fetch revocation information from Google servers!
+    16. A single iOS app for evaluation purposes.
+    17. Uses the test stage
+    18. Custom trusted root is set, to enable generating iOS attestation proofs in software for evaluation purposes.
+    19. This could already be a production value, in preparation for the real iOS app
+    20. This is simply Apple's recommendation plus five minutes offset
```

Comment:
The value described is 600 ... so Apple's recommendation is 5 minutes, which are 300 seconds?